src/pages/EntitiesSearchPage/EntitiesSearchPage.jsx

```diff
@@ -274,6 +294,18 @@ function EntitiesSearch() {
                             >
                                 Reset Search
                             </EuiButton>
+
+                            { disableSaveSearch
```

Comment:
I know it seems odd to conditionally render a tooltip based on button disabled status – this is a workaround to prevent the tooltip from getting stuck on button click, which seems to happen because the button is disabled and then re-enabled after click 😅